otherfiles/requirements-splitter.py

```diff
@@ -0,0 +1,9 @@
+test_req = ""
```

Comment:
Please consider the following:

1. Add coding (`# -*- coding: utf-8 -*-`)
2. Add general docstring (`"""Requirements splitter."""`)